util/MySqlMigrations/Migrations/20231207194013_Send_Add_CipherId_And_ForeignKey.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Bit.MySqlMigrations.Migrations;
+
+/// <inheritdoc />
+public partial class Send_Add_CipherId_And_ForeignKey : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.AlterColumn<string>(
```

Comment:
❌ All this is coming from a drift I think and you need to run all the latest migrations first.